lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -8,15 +8,25 @@ defmodule Apr.Views.CommerceOrderSlackView do
 
   alias Apr.Subscriptions.Subscription
 
-  def render(
-        %Subscription{theme: "fraud"},
-        %{"verb" => verb, "properties" => %{"items_total_cents" => items_total_cents, "mode" => mode}},
-        _routing_key
-      )
-      when items_total_cents < 3_000_00 or verb != "submitted" or mode == "offer",
-      do: nil
+  def render(nil, event, routing_key) do
+    generate_slack_message(event, routing_key)
+  end
+  
+  def render(subscription, event, routing_key) do
+    case {subscription.theme, event["verb"], event["properties"]} do
+      {"fraud", "submitted", %{"mode" => "buy", "items_total_cents" => cents}} when cents >= 3_000_00 ->        
+        generate_slack_message(event, routing_key)
+      {"fraud", "approved", %{"mode" => "offer", "currency_code" => currency_code, "items_total_cents" => cents}} 
+        when cents >= 9_500_00 and currency_code == "EUR" or currency_code == "GBP" ->
```

Comment:
Are we taking a more conservative approach to notify >= 9.5K rather than 10K?

Reply:
Yes! Back in one of our earlier mob sessions Guillaume paired with us and suggested we run the check a little more conservatively

Reply:
Is the logic here going to be `true` when `currency_code == "GBP"` no matter what the amount is?

Reply:
Ah, that's a good point @starsirius. @jpotts244 if nothing else it might be good to wrap the `or` statement in parenthesis just to clarify.

Reply:
Ah, great catch -- I will expose that in a test and fix!